diff --git a/source/components/utilities/utdebug.c b/source/components/utilities/utdebug.c
index f789e686d..20733886b 100644
--- a/source/components/utilities/utdebug.c
+++ b/source/components/utilities/utdebug.c
@@ -166,6 +166,10 @@ static const char           *AcpiGbl_FunctionEntryPrefix = "----Entry";
 static const char           *AcpiGbl_FunctionExitPrefix  = "----Exit-";
 
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdangling-pointer"
+
+
 /*******************************************************************************
  *
  * FUNCTION:    AcpiUtInitStackPtrTrace
@@ -829,4 +833,7 @@ AcpiTracePoint (
 ACPI_EXPORT_SYMBOL (AcpiTracePoint)
 
 
+#pragma GCC diagnostic pop
+
+
 #endif
